### DISCUSSION TITLE
newbee-guide

### DISCUSSION THREAD

**Discussion by giscus** [studyrs/RusttBook#36] [giscus comments]
# newbee-guide

https://guide.rustt.org/newbee-guide.html

**Comment by GiveMe-A-Name** [comment]
> 2. 提交选题并不能加分，只有当选题被翻译者认领后，才能获取加分。因此，现在你需要认领自己提交的那两个选题。

代表我可以自己提交选题，自己进行认领，进行翻译吗？

**Reply by sunface** [reply]
可以的～不过首先，你需要加入 Rustt :D
